src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTraitTest.php

```diff
@@ -528,6 +531,28 @@ public function testGetDoctrine()
 
         $this->assertEquals($doctrine, $controller->getDoctrine());
     }
+
+    public function testGetParameter()
+    {
+        $container = new Container(new FrozenParameterBag(array('foo' => 'bar')));
+        $container->set('parameter_bag', new ContainerBag($container));
+
+        $controller = $this->createController();
+        $controller->setContainer($container);
+
+        if (interface_exists(ContainerBagInterface::class)) {
+            $this->assertSame('bar', $controller->getParameter('foo'));
+
+            return;
+        }
+
+        if (method_exists($this, 'expectException')) {
```

Comment:
You can remove this check for new code. As Symfony 4 requires PHP 7.1+, it will not need to run tests on PHPUnit 4.8 (which is needed for PHP 5.5 and older). So `expectException` can always be used.

Reply:
thanks, we have a bunch of similar checks to remove https://github.com/symfony/symfony/search?utf8=%E2%9C%93&q=setExpectedException&type=